microsoft-edge/progressive-web-apps-chromium/index.md

```diff
@@ -1,235 +1,151 @@
-        :::image type="icon" source="./media/i_security-small.png":::
-    :::column-end:::
-:::row-end:::
-:::row:::
-    :::column:::
-        ### [Network Independent][MDNPwaAdvantagesNetworkIndependent]
-    :::column-end:::
-    :::column:::
-        ### [Progressive][MDNPwaAdvantagesProgressive]
-    :::column-end:::
-    :::column:::
-        ### [Safe][MDNPwaAdvantagesSafe]
-    :::column-end:::
-:::row-end:::
-:::row:::
-    :::column:::
+
+        **[Network Independent][MDNPwaAdvantagesNetworkIndependent]**  
         Works offline and in low-network conditions
     :::column-end:::
     :::column:::
+        :::image type="icon" source="./media/i_progressive-small.png":::
+
+        **[Progressive][MDNPwaAdvantagesProgressive]**  
         Experience scales up (or down) with device capabilities
     :::column-end:::
     :::column:::
+        :::image type="icon" source="./media/i_security-small.png":::
+
+        **[Safe][MDNPwaAdvantagesSafe]**  
         Provides a secure HTTPS endpoint and other user safeguards
     :::column-end:::
 :::row-end:::
 :::row:::
     :::column:::
         :::image type="icon" source="./media/i_responsive-small.png":::
-    :::column-end:::
-    :::column:::
-        :::image type="icon" source="./media/i_link-small.png":::
-    :::column-end:::
-    :::column:::
-        &nbsp;
-    :::column-end:::
-:::row-end:::
-:::row:::
-    :::column:::
-        ### [Responsive][MDNPwaAdvantagesResponsive]
-    :::column-end:::
-    :::column:::
-        ### [Linkable][MDNPwaAdvantagesLinkable]
-    :::column-end:::
-    :::column:::
-        &nbsp;
-    :::column-end:::
-:::row-end:::
-:::row:::
-    :::column:::
+
+        **[Responsive][MDNPwaAdvantagesResponsive]**  
         Adapts to the user's screen size or orientation and input method
     :::column-end:::
     :::column:::
+        :::image type="icon" source="./media/i_link-small.png":::
+
+        **[Linkable][MDNPwaAdvantagesLinkable]**  
         Share and launch from a standard hyperlink
     :::column-end:::
     :::column:::
-        &nbsp;
     :::column-end:::
 :::row-end:::
 
-Build \(or convert\) your existing website to a PWA to enhance engagement with your users. Enhancements include push notifications, app-like integration, and offline support. Continue to build your audience on the open web for users to discover your PWA through search and link-sharing. Best of all, your app is updated using your web server code.
 
-## PWAs on Microsoft Edge (Chromium)
+## PWAs on desktop
 
-When you build a Progressive Web App targeting web standard APIs, your app may be deployed across platforms and devices and take advantage of the device-specific capabilities as available. PWAs in Microsoft Edge \(Chromium\) add the following advantages to your website.
+PWAs aren't only a way for websites to appear on mobile home screens. They're based on standard, cross-browser, web technologies that allow them to be installed and to run everywhere.
```

Comment:
Instead of "aren't only," maybe "are more than"?

Reply:
Agreed and more ... I re-wrote both sentences. Is it an improvement?

Reply:
LGTM